src/tools/convert/topology_helper.py

```diff
@@ -58,7 +58,12 @@ def FillFileBuffer(fname, filebuffer):
     for line in f:
 	if "include" in line and not line[0]==';':
 	    name=(line.split()[1]).strip('\"')
-	    FillFileBuffer(name, filebuffer)
+            try: 
+ 	        FillFileBuffer(name, filebuffer)
+            except IOError:
+                #need to use relative path
+                name = fname.split('/')[0] + '/' + name
```

Comment:
It doesn't seem to be working for me. If I run
`gromacs.read('conf.gro','/full/path/to/topol.top')`
or
`gromacs.read('conf.gro','topol.top')`
and topol.top contains the line
`#include "amber94.ff/forcefield.itp"`
and the file ./amber94.ff/forcefield.itp does exist, then `FillFileBuffer` (the version you suggest) goes looking for
`amber94.ff/amber94.ff/forcefield.itp`
What kind of directory structure are you using your version with? Also, I guess you also changed the line in gromacs.read() where FillFileBuffer is called. What did you put?